### PR TITLE
Deployment take 2

### PR DESCRIPTION
**Pull request by orta**
Will comment inline

### CODE REVIEW

**Comment by orta** [comment]
Merge'd master, should be good to go, last fail was danger unable to find a commit due to a force push.

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by orta** [comment]
:green_apple:

File: fastlane/Fastfile [hunk 1]
```diff
@@ -11,18 +12,79 @@ lane :ship_beta do
...
+        channel: "collector-gmv-ios",
         payload: {
           'Version' => latest_version,
           'What\'s new' => beta_readme
         },
         default_payloads: []
 end
+
+## We want to have a simpler version of Fastlane's match
+## this does it in a pretty reasonable amount of code.
```
Comment (alloy) [inline fastlane/Fastfile]: Perfect :+1: Can we get rid of the `match` gem and its deps?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -11,18 +12,79 @@ lane :ship_beta do
...
+  # prints out the codesigning identities
+  system "security find-identity -v -p codesigning"
+
+  # Install the iOS distribution certificate, -A
+  system "security import #{signing_root}/ios_distribution.cer  -k ~/Library/Keychains/login.keychain -A"
+
+  # Move our provisioning profile in
+  profile_path = File.expand_path("~") + "/Library/MobileDevice/Provisioning Profiles/"
+  destination = profile_path + "/" + app_name + ".mobileprovision"
+  profile = Dir.glob(signing_root + "/profiles/" + app_name + "/*").first
```
Comment (alloy) [inline fastlane/Fastfile]: Better to use things like `File.join` then `+` etc, but no biggie.

**Comment by orta** [comment]
OK, I've removed match, and fui - it wasn't being used. 

I've also updated Danger to this PR https://github.com/KrauseFx/danger/pull/40/ - which makes `--verbose` work - should make debugging DM easier for me.

**Comment by alloy** [comment]
Awesome, can’t wait to deploy!

**Pull request merged**